### PR TITLE
feat(environment): users controlled feature flags

### LINKED ISSUE #5372: Allow users to locally flag features on / off to use bleeding edge features if desired
## The Chore

Could do this using the console in the browser, or could add it to settings.

Barney is keen on this so seems like adding it to settings would be a nicer way of doing it. 

Marked with some appropriate "these features are still in development and are used at your own risk" type warning.

## Tasks

- [ ]
- [ ]

### PR DESCRIPTION
**Pull request by bglownia**
# Related issues 🔗

Closes #5372
# Description ℹ️

A description of what this PR does. This should be non-dev readable. Technical stuff should go in the technical section below.

# Demo 📺

Gif/video/images of new/corrected functionality if applicable

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: libs/environment/src/hooks/use-environment.ts [hunk 1]
```diff
@@ -269,11 +144,16 @@ const testSubscription = (client: Client) => {
   });
 };
 
+export const userControllableFeatureFlags: (keyof FeatureFlags)[] = [
```
Comment (bglownia) [inline libs/environment/src/hooks/use-environment.ts:147]: reset to empty array before merge

File: apps/trading/components/settings/settings.tsx [hunk 2]
```diff
@@ -98,6 +124,11 @@ export const Settings = () => {
           <dd className="break-words">{process.env.GIT_COMMIT}</dd>
         </dl>
       </SettingsGroup>
+      <SettingsGroup inline={false} label={t('Experimental features')}>
```
Comment (bglownia) [inline apps/trading/components/settings/settings.tsx:127]: remove before merge

File: apps/trading/.env [hunk 3]
```diff
@@ -20,8 +20,8 @@ NX_WALLETCONNECT_PROJECT_ID=fe8091dc35738863e509fc4947525c72
 
 # Cosmic elevator flags
 NX_SUCCESSOR_MARKETS=true
-NX_STOP_ORDERS=true
+NX_STOP_ORDERS=false
```
Comment (bglownia) [inline apps/trading/.env:23]: rollback before merge

**Review by bwallacee — COMMENTED** [review]
Can we add a unit test for when feature flag is set to false as well?
Also should we log out to sentry what flags are getting enabled.
e.g. if we get an error in console and log to sentry we should also log what flags were enabled (to help debugging etc)

**Review by mattrussell36 — APPROVED** [review]
Looks good. One comment about tidying up the dupcliate mock, also I agree using the switch would be nicer.

**Review by bwallacee — APPROVED** [review]

**Pull request merged**